polymer.json

```diff
@@ -0,0 +1,19 @@
+{
+  "entrypoint": "app/1.0/index.html",
+  "shell": "app/elements/pw-shell.html",
+  "fragments": [
+    "app/elements/pw-shell.html",
```

Comment:
Since it's already in `"shell": "app/elements/pw-shell.html",`, then we don't need to put it in the fragments.